application/src/main/java/org/opentripplanner/graph_builder/module/DirectTransferGenerator.java

```diff
@@ -247,5 +269,114 @@ private NearbyStopFinder createNearbyStopFinder() {
     }
   }
 
+  private void createPathTransfer(
+    StopLocation from,
+    StopLocation to,
+    NearbyStop sd,
+    Map<TransferKey, PathTransfer> distinctTransfers,
+    StreetMode mode
+  ) {
+    TransferKey transferKey = new TransferKey(from, to, sd.edges);
+    PathTransfer pathTransfer = distinctTransfers.get(transferKey);
+    if (pathTransfer == null) {
+      // If the PathTransfer can't be found, it is created.
+      distinctTransfers.put(
+        transferKey,
+        new PathTransfer(from, to, sd.distance, sd.edges, EnumSet.of(mode))
+      );
+    } else {
+      // If the PathTransfer is found, a new PathTransfer with the added mode is created.
+      distinctTransfers.put(transferKey, pathTransfer.withAddedMode(mode));
+    }
+  }
+
+  /**
+   * This method parses the given transfer parameters into a transfer configuration and checks for invalid input.
+   */
+  private TransferConfiguration parseTransferParameters(NearbyStopFinder nearbyStopFinder) {
+    List<RouteRequest> defaultTransferRequests = new ArrayList<>();
+    List<RouteRequest> carsAllowedStopTransferRequests = new ArrayList<>();
+    List<RouteRequest> flexTransferRequests = new ArrayList<>();
+    HashMap<StreetMode, NearbyStopFinder> defaultNearbyStopFinderForMode = new HashMap<>();
+    /* These are used for calculating transfers only between carsAllowedStops. */
```

Comment:
Use normal comments inside methods.

Reply:
These were from a previous commit. I changed them in this file everywhere I could find them